static/src/javascripts/projects/common/modules/ui/images.js

```diff
@@ -27,7 +27,7 @@ function (
                 },
                 'ui:images:lazyLoaded': function (context) {
                     picturefill({
-                        elements: [context]
+                        elements: context ? [context] : null
```

Comment:
Passing in a context here expects an array of images as per output from picturefill's [getAllElements](https://github.com/guardian/frontend/blob/Sustain_10610-related-images/static/src/javascripts/projects/common/utils/picturefill.js#L383) - do we / should we have a util method that can return this based on a context? Rather than modify picturefill directly, maybe we can duplicate that method as a Guardian util...?